packages/local_auth/android/build.gradle

```diff
@@ -48,6 +48,6 @@ android {
 
 dependencies {
     api "androidx.core:core:1.1.0-beta01"
-    api "androidx.biometric:biometric:1.0.0-alpha04"
+    api "androidx.biometric:biometric:1.0.0-beta01"
```

Comment:
In general we've been trying to guard dependency upgrades behind breaking version change bumps, out of an abundance of caution for when there's breaking changes in the dependency itself.

Reply:
Both androidx.biometrics and this plugin is not 1.0 yet. So breaking changes are expected. Having said that, I am happy to bump version to 0.6. Would that be sufficient?

Reply:
0.6 SGTM, thanks. The semantic versioning pub convention for plugins under `1.0.0` is `0.major.minor+patch` instead of `major.minor.patch`, so `0.6.0` would be the breaking change update here.